epics/mutationEpics.js

```diff
@@ -15,7 +15,13 @@ export default function mutationEpics(resource) {
       path = path && path.replace(/[\/].*$/g, '');  // eslint-disable-line no-useless-escape
 
       const name = resource.name;
-      const meta = Object.assign({}, action.meta, { path, name });
+      const meta = {
+        ...action.meta,
+        originatingActionType: action.type,
```

Comment:
Would not be `type` be also good here?

Reply:
Now I do not think it is a great idea, as `action.meta.type` is a kind of misleading.

Reply:
I don't think so, because there's already a `type` is at the top level of the `action` object which refers to the type of the _refresh_ action being generated. To avoid confusion, I was very literal with naming this.